---
title: "The architecture of brain co-expression reveals the brain-wide basis of disease susceptibility"

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here 
# and it will be replaced with their full name and linked to their profile.
authors:
- Christopher L Hartl
- Gokul Ramaswami
- William Pembroke
- admin
- Princy Parsana
- Sandrine Muller
- Greta Pintacuda
- Kasper Lage
- Alexis Battle
- Daniel H Geschwind

# # Author notes (optional)
# author_notes:
# - "Corresponding Author"
# - "Corresponding Author"

date: "2020-03-05T00:00:00Z"
doi: ""

# Schedule page publish date (NOT publication's date).
publishDate: "2017-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["3"]

# Publication name and optional abbreviated publication name.
publication: In *bioRxiv*
publication_short: In *bioRxiv*

abstract: Gene networks have proven their utility for elucidating transcriptome structure in the brain, yielding numerous biological insights. Most analyses have focused on expression relationships within a circumspect number of regions â€“ how these relationships vary across a broad array of brain regions is largely unknown. By leveraging RNA-sequencing in 864 samples representing 12 brain regions in a cohort of 131 phenotypically normal individuals, we identify 12 brain-wide, 114 region-specific, and 50 cross-regional co-expression modules. We replicate the majority (81%) of modules in regional microarray datasets. Nearly 40% of expressed genes fall into brain-wide modules corresponding to major cell classes and conserved biological processes. Region-specific modules comprise 25% of expressed genes and correspond to region-specific cell types and processes, such as oxytocin signaling in the hypothalamus, or addiction pathways in the nucleus accumbens. We further leverage these modules to capture cell-type-specific lncRNA and gene isoforms, both of which contribute substantially to regional synaptic diversity. We identify enrichment of neuropsychiatric disease risk variants in brain wide and multi-regional modules, consistent with their broad impact on cell classes, and highlight specific roles in neuronal proliferation and activity-dependent processes. Finally, we examine the manner in which gene co-expression and gene regulatory networks reflect genetic risk, including the recently framed omnigenic model of disease architecture.

# # Summary. An optional shortened abstract.
# summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags: []

# Display this page in the Featured widget?
featured: false

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

url_pdf: 'paper/brain_modules.pdf'
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# image:
#   caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/pLCdAaMFLTE)'
#   focal_point: ""
#   preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
# projects:
# - example

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
# slides: crossmap
---
