---
title: Tissue-specific transcriptome-wide networks reflect joint regulation of alternative splicing and gene expression

event: American Society of Human Genetics 2015 Annual Meeting
event_url: https://www.ashg.org/meetings/

location: Baltimore Convention Center
address:
  street: 1 West Pratt Street
  city: Baltimore
  region: MD
  postcode: '21201'
  country: United States

# summary: An example talk using Wowchemy's Markdown slides feature.
abstract: "Alternative splicing plays an important role determining cellular function in eukaryotes; its causal role in human diseases including cancer is well established. However, the regulation of alternative splicing is not fully understood, including the factors that lead to tissue-specific control of alternative splicing. Unraveling the regulation of alternative splicing in the form of a network has the potential to elucidate complex biological mechanisms. Until recently, well-studied regulatory network inference methods predominantly used total gene expression data. However, with the advent of RNA-sequencing, we are now able to simultaneously quantify a diverse range of transcription phenotypes, including alternative splicing, non-coding transcripts, and allele-specific expression. We can discover new regulatory relationships by using these phenotypes during network construction. Here we build transcriptome-wide networks (TWN) over both gene expression and alternative splicing from RNA-seq data using sparse Gaussian Markov random fields. We identify multiple types of candidate regulatory relationships where i) gene expression regulates alternative splicing, ii) alternative splicing regulates gene expression, iii) gene expression of a gene regulates that of other genes, and iv) alternative splicing of a gene regulates that of other genes. We learn TWNs for 23 different human tissues using RNA-seq data from the GTEx project. Our networks are enriched with edges between known transcription factors and their target genes, demonstrating our model's effectiveness to find true regulatory relationships. Further, candidate splicing regulator genes identified by learned TWNs include many known RNA-binding proteins (RBPs), reflecting the role of RBPs in splicing. We integrate expression quantitative trait loci and allele-specific expressions to validate potentially causal relationships in TWNs, and demonstrate improved power to detect trans expression quantitative trait loci. Numerous tissue-specific regulatory relationships are identified by the TWNs, and are informative in characterizing context-specific behavior of regulatory elements. Our transcriptome-wide networks provide an opportunity to understand tissue-specific regulatory mechanisms for both alternative splicing and gene expression."

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
date: "2015-10-07T9:00:00Z"
date_end: "2015-10-07T17:00:00Z"
all_day: false

# Schedule page publish date (NOT talk date).
publishDate: "2017-01-01T00:00:00Z"

authors: []
tags: []

# Is this a featured talk? (true/false)
featured: false

image:
  caption: ''
  focal_point: Right

# links:
# - icon: twitter
#   icon_pack: fab
#   name: Follow
#   url: https://twitter.com/georgecushen
url_code: ""
url_pdf: ""
url_slides: "talk/ASHG-2015.pptx"
url_video: ""

# Markdown Slides (optional).
#   Associate this talk with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
# slides: example

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
# projects:
# - example
---

